fineract-core/src/main/java/org/apache/fineract/infrastructure/security/utils/SQLCommandCondition.java

```diff
@@ -0,0 +1,5 @@
+package org.apache.fineract.infrastructure.security.utils;
```

Comment:
Please add the ASF license! (you can copy from an another file)

Reply:
@adamsaghy Thanks for review. Please check the changes. 🙇